engine/admin/ambassador_pattern_linking.md

```diff
@@ -151,4 +150,4 @@ case `192.168.1.52:6379`.
     	apk add socat && \
     	rm -r /var/cache/
 
-    CMD	env | grep _TCP= | (sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \&/' && echo wait) | sh
+    CMD	env | grep _TCP= | (sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/socat -t 100000000 TCP4-LISTEN:\1,fork,reuseaddr TCP4:\2:\3 \&/' && echo wait) | sh
```

Comment:
@thaJeztah yes, thinking about it, I should have discarded those last-line-changes before committing. Is there some documentation about this for the docs case? Are these trailing new lines required on this repo?